Internal/Debug App/Tests/Unit Tests/Primer/PrimerBancontactCardDataManagerTests.swift

```diff
@@ -13,6 +13,8 @@ import XCTest
 
 class PrimerBancontactCardDataManagerTests: XCTestCase {
     
+    private static let expectationTimeout = 1.0
```

Comment:
Worth making global? Thinking we can override for individual suites or tests later if they need it and a global makes it easier to change in one place when an Xcode release causes simulator-wide overheads (might be overthinking this :-D)

Reply:
It did cross my mind as it is the same in two places right now, but I held off as there are expectation timeouts of various lengths throughout the tests, all depends on the work going on in the test.

Feels like a per-file value to me

Reply:
Yikes, agreed - another one to note for discussion (600s !?)